modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js

```diff
@@ -79,23 +79,31 @@ const SidebarPanelInfoView = ({
 
 	return (
 		<>
-			<Sidebar.Header subtitle={subType} title={title}>
-				{versions.map((version) => (
-					<div key={version}>
-						<ClayLabel displayType="info">
-							{`${Liferay.Language.get('version')} ${
-								version.version
-							}`}
-						</ClayLabel>
+			<Sidebar.Header title={Liferay.Language.get('content-info')} />
 
-						<ClayLabel displayType={version.statusStyle}>
-							{version.statusLabel}
-						</ClayLabel>
+			<Sidebar.Body>
+				<div className="mb-4 sidebar-dl sidebar-section">
+					<div className="component-title text-truncate-inline">
+						<span className="text-truncate">{title}</span>
 					</div>
-				))}
-			</Sidebar.Header>
 
-			<Sidebar.Body>
+					<p className="component-subtitle">{subType}</p>
+
+					{versions.map((version) => (
+						<div key={version}>
+							<ClayLabel displayType="info">
+								{`${Liferay.Language.get('version')} ${
+									version.version
+								}`}
```

Comment:
Totally subjective, but might look cleaner without the interpolation and the template string:

ie.

```js
<ClayLabel displayType="info">
    {Liferay.Language.get('version')} {version.version}
</ClayLabel>
```

instead of:

```js
<ClayLabel displayType="info">
    {`${Liferay.Language.get('version')} ${
        version.version
    }`}
</ClayLabel>
```

One [gotcha about JSX](https://www.javascriptstuff.com/how-whitespace-works-in-jsx/):

> In general, the whitespace at the beginning and end of each line is removed, and newlines are removed.

Which is why I wrote what I did and not:

```js
<ClayLabel displayType="info">
    {Liferay.Language.get('version')}
    {version.version}
</ClayLabel>
```

(That looks better in the source code, but gets mashed together when rendered).

Reply:
formater converts this
```
<ClayLabel displayType="info">
    {Liferay.Language.get('version')} {version.version}
</ClayLabel>
```

into this

```
ClayLabel displayType="info">
    {Liferay.Language.get('version')}
    {version.version}
</ClayLabel>
```

Reply:
Which formatter? That's a breaking change, so Prettier wouldn't do that; at most it would do [this](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAwgGwIYE8Ay2ARnJgAQAmAlgM4AOOuAKrnXALwA6IVUAZhG4A+TlDLiywfFT5wATngB0hKAHMArtlVxF2mAAoA5ADd5NKtEMBKAL6TTc89EUOnUG6NQB6LHkIlMIRAAGhAIOhgLKBpkUGw5OQgAdwAFeIQYlGxMJLwY0KIFMABrOBgAZTpsMF5VZBg5dThQgAsYAFtMAHUWqnh6arhyjL6qYz7cZHAafJ5o+RgUhVV27GQ+bJpmkAArGgAPACEi0orsdrhpKDh1ze29-fLazDgARXUIeFvMLdCqx3kUyIxFIIRAdDkvBgXSoFBgLWQAFYAMx-RJbLoKOhTCFwLZyUxggCOH3gS3CmRA2BoAFprnAKAywXI4CSqCyllpVt9fiAtu0qPVGttzGoXu9PjckA0mqEYMQYXCEUgACxyhRUTC1dAQdrclB4xFg9RbJjETIy7bGJoASSgjNg5TAkIiAEF7eUYLgXjy4DYbEA):

```
<ClayLabel displayType="info">
	{Liferay.Language.get('version')}{' '}
	{version.version}
</ClayLabel>
```

Reply:
You're totally right, didn't notice the `{' '}` part